showcases/next-showcase/package.json

```
@@ -13,7 +13,7 @@
     "lint": "next lint"
   },
   "dependencies": {
-    "@db-ui/foundations": "file:./../../packages/foundations",
+    "@db-ui/foundations": "file:../../packagesi/foundations",
```

@nmerget Is there a typo in the path ("packages**i**")?
```suggestion
    "@db-ui/foundations": "file:../../packages/foundations",
```

---

fixed